jobserver/views/orgs.py

```diff
@@ -22,16 +25,35 @@ def get_success_url(self):
 
 
 class OrgDetail(DetailView):
-    model = Org
-    slug_url_kwarg = "org_slug"
     template_name = "org_detail.html"
 
-    def get_context_data(self, **kwargs):
-        projects = self.object.projects.order_by("name")
-
-        context = super().get_context_data(**kwargs)
-        context["projects"] = projects
-        return context
+    def get(self, request, *args, **kwargs):
+        slug = self.kwargs["org_slug"]
+
+        try:
+            org = Org.objects.get(slug=slug)
+        except Org.DoesNotExist:
+            # we used to serve up Workspaces at the root URL but have moved
+            # them under their relevant Orgs & Projects.  So attempt to
+            # redirect requests to a valid Workspace here for a while.
```

Comment:
How long do we plan to support this?  Is it likely that any workspaces URLs are relied on somewhere?

Reply:
Unsure currently, I've vaguely thought somewhere in the region of a month or two, but it's possible this could live here permanantely depending on how we feel about the far older workspace links being valid.